backend/src/main/java/de/neuefische/backend/appUser/AppUser.java

```diff
@@ -11,5 +11,7 @@ public class AppUser {
     private String id;
     private String username;
     private String password;
-   // private String iban;
+    private int accountNumber;
```

Comment:
Would you be better of using a long instead of an int here?

Reply:
long is not needed because the account number will only be maximum 2 numbers and the "DE 99 9009 9009 0000" will be hardcoded in frontend or somehow implemented later. It is not that easy because the number is large and it contains a String also.

Reply:
Fair enough!
*Check*